### PR TITLE
jp - initial commit for Save Image Link

### LINKED ISSUE #2: Save Image Link
#### As a user I want to save a link to an image.

Acceptance criteria:
- [x] The link is entered through a form.
- [x] After the form is saved, the link is persisted in the database.

Dependencies:
- Deployment Environment

### PR DESCRIPTION
**Pull request by jeremyjpark**
Closes #2 

Requesting code review for the second issue

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,6 +28,9 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use puma as the app server
 gem 'puma'
 
+# Use AutoLink to allow text to be urls if needed. Caused by Rails 4 deprecation of auto_link
+gem 'rails_autolink'
```
Comment (scuendet) [inline Gemfile]: Why do we need to use this gem here? It seems like this was removed from rails 3.1 and is only provided for people migrating: https://github.com/tenderlove/rails_autolink

File: Gemfile.lock [hunk 2]
```diff
@@ -230,5 +233,8 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
 
+RUBY VERSION
```
Comment (scuendet) [inline Gemfile.lock]: Did you add that manually?

File: app/controllers/links_controller.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
+class LinksController < ApplicationController
+  before_action :set_link, only: [:show, :edit, :update, :destroy]
+
+  # GET /links
+  # GET /links.json
```
Comment (scuendet) [inline app/controllers/links_controller.rb]: This file looks like it was generated by a scaffold. It's fine to use a scaffold, but can you remove the methods you don't use in your app for the moment? Also, I'm not sure these comments above the routes help much...

File: Gemfile.lock (hunk 2 above)
Comment (jeremyjpark) [inline Gemfile.lock]: I don't think I added this manually; I thought that it was a side-effect of adding the rails_autolink gem?

**Pull request merged**